packages/cloud-cognitive/src/components/Datagrid/Datagrid/DatagridRow.js

```diff
@@ -43,6 +43,30 @@ const DatagridRow = (datagridState) => {
           `${blockClass}__carbon-row-expanded-hover-active`
         );
       }}
+      onKeyUp={(event) => {
```

Comment:
As discussed over slack, try just adding a conditional class to the row if the row index is equal to the `activeCellId` row

Reply:
@matthewgallo The changes you suggested have been implemented. Thanks for the feedback.